content/en/blog/wireshark.md

```diff
@@ -1,21 +1,22 @@
 ---
 draft: true
-spelling: cSpell:ignore addressbook Huang pcapng Qiangxiong subdissectors tcpdump Wireshark
+spelling: cSpell:ignore addressbook Chalin Huang pcapng Qiangxiong subdissectors tcpdump Wireshark
 title: Analyzing gRPC messages using Wireshark
-date: 2020-12-04
-author:
-  name: Huang Qiangxiong
+date: 2021-01-18
+authors:
+- name: Huang Qiangxiong
   link: https://github.com/huangqiangxiong
+- name: Patrice Chalin (editor)
+  link: https://github.com/chalin
```

Comment:
Given the extensive revisions made to the post, I've added myself as an editor. Let me know if you're ok with that.